include/wx/debug.h

```diff
@@ -328,6 +335,10 @@ extern WXDLLIMPEXP_BASE void wxOnAssert(const char *file,
 
     #define wxASSERT(cond)
     #define wxASSERT_MSG(cond, msg)
+
+    #define wxASSERT_NOERR_MSG(errorCode, msg)
+    #define wxASSERT_NOERR(errorCode)
+
```

Comment:
Just FYI: I'm removing those as this looks like a leftover: these macros have no non-trivial definitions anywhere.

Reply:
yes, sorry, leftovers, I first wanted to use asserts, but then I realized, that they are in some parts used directly with the OS call itself, so CHECK was more appropriate